packages/input-checkbox/src/Checkbox.js

```diff
@@ -18,20 +13,8 @@ const createCheckbox = ({ isSwitch }) =>
     };
 
     render() {
-      const {
-        type,
```

Comment:
`type` should still be removed from `inputProps`. Otherwise the `type` could be wrongly passed to `input`

Reply:
`const { label, checked, onMouseUp, onChange, ...inputProps } = this.props;`

`<Checkbox type='number' />` will pass `type` to `<input type="checkbox" {...inputProps} className="hiddenCheckbox" />`

Reply:
everything that is not declared before the spread is part of `inputPropos`

Reply:
I am in favor of removing that type however, because it doesnt make much sense to pass a type attribute to a Checkbox component, we don't have instances of such combo neither in webapp nor farmblocks, and may not happen (specially considering this prop is not documented). It is different from the case of the input text component, that someone could want to forward the type="email" type="phone" or something like that to the underlining html input, in this case the underlining html input will be always type="checkbox"

Reply:
Just to document what we talked about.
Let's just change the order of the properties from 
`<input type="checkbox" {...inputProps}  className="hiddenCheckbox" />` 
to 
`<input {...inputProps} type="checkbox" className="hiddenCheckbox" />`